msl_expressions/autogenerated/src/Plans/Behaviours/Intercept.cpp

```diff
@@ -128,7 +130,7 @@ namespace alica
             }
         }
 
-        predBall->alloToEgo(*predPos);
+        auto egoPredBall = predBall->alloToEgo(*predPos);
```

Comment:
There is already an existing ticket for improvement of this behaviour in our Jira. 
I'd suggest we keep the code from the tournament branch because we fixed this obvious bug and didn't delete anything that could be valuable.